unified-ai-platform-samples/README.md

```diff
@@ -0,0 +1,6 @@
+# Google Cloud AI Platform (Unified) Products
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+
+Welcome to the [AI Platform (Unified)](https://cloud.google.com/ai-platform/docs/) samples repository.
```

Comment:
1. Change URL to https://cloud.google.com/ai-platform-unified/docs/
2. Super-nit: I would maybe remove the word "repository" since it is not technically a separate repository. "Welcome to the AI Platform (Unified) samples." But this is very nit-picky and not important. 😄